app/views/assistants/_form.html.erb

```diff
@@ -1,10 +1,9 @@
-<h1>Add a new assistant to <%= @course.title %></h1>
+<h4>Add a new assistant (give username)</h4>
```

Comment:
Ah, never mind, I was thinking of the form as an entire page but it isn't. Ignore my silliness.